recipes/pygeohash/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pygeohash" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 750e51643e743eabd065a84fc8c1912c5843b648143137919e4a9776366d921e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  files:
+    - tests/test_pygeohash.py
```

Comment:
It looks like you’re not using this file when testing?

Reply:
It is using the file and I do noticed logged outputs, but can double check on that

Reply:
You probably need to add a `commands` section to run whichever testing framework you use (adding it as a test dependency if required).

Reply:
You're currently not using this file. You should either remove the `files` section of `test` or add:
```yaml
  requires:
    - nose
  commands:
    - nosetests
```

Reply:
Got it! just fixed that